### PR TITLE
Fix icons doubling and WebOS support

### LINKED ISSUE #942: Double "Play" button when manually starting task
**Describe the bug**
When manually starting a task the "play" buttons of the other tasks double most of the time.
There are different flavours of this:
* All "Play" buttons double after starting a task
* Play button flashes up in front of the "Stop" button of the started task and after that everything is as it should
* Additional "Play" Button flashes up before all other "Play" buttons and after that everything is as it should
* The additional buttons hide when you start another task

**To Reproduce**
1. Go to *Scheduled Tasks*
2. Start a task
3. Watch the screen

**Expected behavior**
Don't show doubled button

**Screenshots**
Screenshot_2020-03-13 Jellyfin

**System:**
 - OS: Arch, Windows 10
 - Browser: Firefox, Opera
 - Jellyfin Version: 10.5.0 nightly

### PR DESCRIPTION
**Pull request by dmitrylyzo**
**Changes**
* Fix icon doubling in "Dashboard / Scheduled Tasks"
* Fix library icons on WebOS 4
* Fix slideshow button icons in WebOS 4
* Fix emby-collapse icon in WebOS 4

**Issues**
https://github.com/jellyfin/jellyfin-web/pull/898#issuecomment-596815235
Fixes #942

**Screenshots**
_Fix library icons on WebOS 4_
library-1
library-2
_Alignment issue note below_

_Fix slideshow button icons in WebOS 4_
slideshow

_Fix emby-collapse icon in WebOS 4_
filter

**Alignment note**
_In Chrome with TV layout, the same._
It could be fixed by `display: flex` and, actually, makes better alignment in Desktop. I tried to remove this
https://github.com/jellyfin/jellyfin-web/blob/fcc5bed8632c2db4df57f270a90c308ee637f4c0/src/components/cardbuilder/card.css#L193-L194
library-3
But I'm not sure if this affects other cards badly. So, any suggestions?

**More icons**
I skipped some icons, as they do not appear in TV layout (for now) or not use underscore.
https://github.com/jellyfin/jellyfin-web/blob/fcc5bed8632c2db4df57f270a90c308ee637f4c0/src/components/activitylog.js#L21
https://github.com/jellyfin/jellyfin-web/blob/fcc5bed8632c2db4df57f270a90c308ee637f4c0/src/components/cardbuilder/cardBuilder.js#L1507
https://github.com/jellyfin/jellyfin-web/blob/fcc5bed8632c2db4df57f270a90c308ee637f4c0/src/components/listview/listview.js#L163
https://github.com/jellyfin/jellyfin-web/blob/fcc5bed8632c2db4df57f270a90c308ee637f4c0/src/components/userdatabuttons/userdatabuttons.js#L33
https://github.com/jellyfin/jellyfin-web/blob/fcc5bed8632c2db4df57f270a90c308ee637f4c0/src/controllers/auth/selectserver.js#L21
https://github.com/jellyfin/jellyfin-web/blob/fcc5bed8632c2db4df57f270a90c308ee637f4c0/src/controllers/medialibrarypage.js#L270
https://github.com/jellyfin/jellyfin-web/blob/fcc5bed8632c2db4df57f270a90c308ee637f4c0/src/controllers/medialibrarypage.js#L283
https://github.com/jellyfin/jellyfin-web/blob/fcc5bed8632c2db4df57f270a90c308ee637f4c0/src/scripts/librarymenu.js#L450
https://github.com/jellyfin/jellyfin-web/blob/fcc5bed8632c2db4df57f270a90c308ee637f4c0/src/scripts/librarymenu.js#L577

### CODE REVIEW

File: src/controllers/dashboard/scheduledtasks/scheduledtasks.js [hunk 1]
```diff
@@ -90,16 +90,23 @@ define(["jQuery", "loading", "events", "globalize", "serverNotifications", "huma
         return html;
     }
 
+    function setTaskButtonIcon(button, icon) {
+        var inner = button.querySelector("i");
+        inner.classList.remove("stop");
+        inner.classList.remove("play_arrow");
```
Comment (MrTimscampi) [inline src/controllers/dashboard/scheduledtasks/scheduledtasks.js]:
```suggestion
        inner.classList.remove("stop", "play_arrow");
```

This should work as well and is slightly cleaner (I'm not sure how it acts when one of the classes isn't found though, so better test it)

File: src/components/slideshow/slideshow.js [hunk 2]
```diff
@@ -201,14 +201,16 @@ define(['dialogHelper', 'inputManager', 'connectionManager', 'layoutManager', 'f
         function onAutoplayStart() {
             var btnSlideshowPause = dlg.querySelector('.btnSlideshowPause i');
             if (btnSlideshowPause) {
-                btnSlideshowPause.innerHTML = "pause";
+                btnSlideshowPause.classList.remove("play_arrow");
+                btnSlideshowPause.classList.add("pause");
```
Comment (MrTimscampi) [inline src/components/slideshow/slideshow.js]:
```suggestion
                btnSlideshowPause.classList.replace("play_arrow", "pause");
```

File: src/components/slideshow/slideshow.js [hunk 3]
```diff
@@ -201,14 +201,16 @@ define(['dialogHelper', 'inputManager', 'connectionManager', 'layoutManager', 'f
...
             }
         }
 
         function onAutoplayStop() {
             var btnSlideshowPause = dlg.querySelector('.btnSlideshowPause i');
             if (btnSlideshowPause) {
-                btnSlideshowPause.innerHTML = "&#xE037;";
+                btnSlideshowPause.classList.remove("pause");
+                btnSlideshowPause.classList.add("play_arrow");
```
Comment (MrTimscampi) [inline src/components/slideshow/slideshow.js]:
```suggestion
                btnSlideshowPause.classList.replace("pause", "play_arrow");
```

File: src/components/slideshow/slideshow.js (hunk 2 above)
Comment (dmitrylyzo) [reply src/components/slideshow/slideshow.js]: have to be polyfilled

Comment (MrTimscampi) [reply src/components/slideshow/slideshow.js]: We could use [DOM4](https://www.npmjs.com/package/dom4) to polyfill it, as well as a bunch of other things.

**Comment by dkanada** [comment]
@MrTimscampi feel free to tag anything you think is worthy of the next release. We can always remove it later, but adding it will make sure someone looks at it.

**Comment by dkanada** [comment]
@dmitrylyzo is this ready for more reviews?

**Comment by dmitrylyzo** [comment]
> @dmitrylyzo is this ready for more reviews?

If we aren't going to add `dom4` (as in @MrTimscampi 's suggestion), then PR is ready.

**Pull request merged**